splicemachine/notebook.py

```diff
@@ -41,9 +41,15 @@ def hide_toggle(toggle_next=False):
 
     return HTML(html)
 
-def get_mlflow_ui():
-    display(HTML('<font size=\"+1\"><a target=\"_blank\" href=/mlflow>MLFlow UI</a></font>'))
-    return IFrame(src='/mlflow', width='100%', height='500px')
+def get_mlflow_ui(experiment_id=None, run_id=None):
+    if run_id and not experiment_id:
+        raise Exception('If you are passing in a run id, you must also provide an experiment id!')
+    experiment_id = experiment_id or 0
+    mlflow_url = '/mlflow/#/experiments/{}'.format(experiment_id)
+    if run_id:
+        mlflow_url += '/runs/{}'.format(run_id)
+    display(HTML('<font size=\"+1\"><a target=\"_blank\" href={}>MLFlow UI</a></font>'.format(mlflow_url)))
```

Comment:
yuck

Reply:
that's why we made it a notebook shortcut